packages/evolution-common/src/services/widgets/PathConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { ValidationFunction } from './WidgetConfig';
+import { ParserFunction } from '../parsers/ParserFunction';
+
+/**
+ * Path config include custom admin validations,
```

Comment:
This does not explain the why of this type:

"This type describes the paths in the responses field, with data for validation and export". Keep the rest of the description after.

Reply:
added